cmd/shfmt/Dockerfile

```diff
@@ -1,9 +1,18 @@
-FROM golang:1.13.5-alpine3.10
+FROM golang:1.13.5-alpine3.10 as build
 
 WORKDIR /src
 COPY . .
 RUN CGO_ENABLED=0 go build -ldflags '-w -s -extldflags "-static"' ./cmd/shfmt
 
-FROM busybox:1.31.1-musl
-COPY --from=0 /src/shfmt /bin/shfmt
+FROM busybox:1.31.1-musl AS busybox
```

Comment:
I think if we start duplicating images, we might as well make this a proper distro - alpine linux. Just pick the latest tag they've published, and fix all the names elsewhere.

Reply:
Done